components/admin_console/admin_settings.jsx

```diff
@@ -122,6 +127,40 @@ export default class AdminSettings extends React.Component {
         return n;
     };
 
+    getConfigValue(config, path) {
+        const pathParts = path.split('.');
+
+        return pathParts.reduce((obj, pathPart) => {
+            if (!obj) {
+                return null;
```

Comment:
Would it make sense to just `return;` here, so as to implicitly return `undefined` instead of `null`, matching the semantics of `obj.key_that_doesnt_exist`?

Reply:
I don't think we need to match that behaviour exactly since I think we treat undefined and null the same in almost every circumstance. I usually default to null in these cases out of habit